readthedocs/templates/profiles/private/delete_account.html

```diff
@@ -9,12 +9,31 @@
 {% block edit_content_header %} {% trans "Delete Account" %} {% endblock %}
 
 {% block edit_content %}
+
+  {% block delete-warning %}
+    <p>
+      <p>
+        <strong>All projects where you are the only owner will be deleted.</strong>
```

Comment:
Untranslated string here. Also is this the messaging we'll give to commercial users?